CHANGELOG.md

```diff
@@ -6,6 +6,53 @@
 ### Enhancements
 * None
 
+### Fixed
+* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
+* None
+
+### Compatibility
+* React Native >= v0.71.4
+* Realm Studio v15.0.0.
+* File format: generates Realms with format v24 (reads and upgrades file format v10).
+
+### Internal
+<!-- * Either mention core version or upgrade -->
+<!-- * Using Realm Core vX.Y.Z -->
+<!-- * Upgraded Realm Core from vX.Y.Z to vA.B.C -->
+
+## 12.10.0-rc.0 (2024-05-31)
+
+### Enhancements
+* A `counter` presentation data type has been introduced. The `int` data type can now be used as a logical counter for performing numeric updates that need to be synchronized as sequentially consistent events rather than individual reassignments of the number. ([#6694](https://github.com/realm/realm-js/pull/6694))
```

Comment:
```suggestion
* A `counter` presentation data type has been introduced. The `int` data type can now be used as a logical counter for performing numeric updates that need to be synchronized as sequentially consistent events rather than individual reassignments of the number. For a high level introduction, read about [counters in the documentation](https://www.mongodb.com/docs/atlas/app-services/sync/details/conflict-resolution/#counters). ([#6694](https://github.com/realm/realm-js/pull/6694))
```

Reply:
Thanks, good link! (I update the suggestion slightly to add another link as well) 👍